README.md

```diff
@@ -45,11 +49,28 @@ environment variable to include the jar which includes all dependencies:
 BF_CP=target/OMEZarrReader-with-dependencies.jar showinf -nopix your.ome.zarr/.zattrs
 ```
 
-## Known Issues/TODO list
-- Currently working on packaging, discovered issue when connecting to S3 using packaged jar
-- S3 File System Store is likely not ideal sceanrio, other options to be investigated
-- S3 access currently very inefficient
-- Odd issue with data being lost when decompressing bytes in jzarr, an ugly hack is currently in place
-- Identification of S3 location needs updating
-- Refactor code to remove duplication
-- Parse colours for labels
+## Reader specific options
+
+The OMEZarrReader has a number of reader specific options in version 0.4.0 which can be used to customise the reader behaviour. This options can be used in the same manner as the reader options for Bio-Formats outlined [here](https://bio-formats.readthedocs.io/en/latest/formats/options.html#usage). 
+
+The new default behaviour of the `omezarr.include_labels` option introduced in v0.4.0 represents a change in behaviour from the v0.3 releases. Previously any Zarr arrays found in the labels folder would by default be represented as an additional image series. With the current default settings, Zarr arrays in the labels folder will no longer be included in the list of image series. Changing this setting to `true` will revert to the previous behaviour. 
+
+**Note:** If you had imported data with labels using version v0.3 or earlier then you will need to ensure that the `omezarr.include_labels` option is set to true. You can do this either via the API or by adding a `bfoptions` file to the fileset. In the case of OMERO this will require running psql commands to update the database to include the new `bfoptions` file.
```

Comment:
Agreed that I think this is edge-case and don't expect anyone to need this but...

Is there any option to do this via the API when in OMERO? This paragraph is only about OMERO, right?
Maybe start with `"If you had imported data with labels into OMERO...`.
Then you wouldn't need "In the case of OMERO..."?
Also I wonder if you should say "Please contact us on image.sc...." or something like that, otherwise it reads as if a user should know what psql commands to run.

Reply:
Yeah the API would not be for OMERO, I will make that paragraph OMERO specific and add the image.sc reference